dask_sphinx_theme/layout.html

```diff
@@ -0,0 +1,56 @@
+{% extends "sphinx_rtd_theme/layout.html" %}
+
+{% set css_files = css_files + ["_static/css/explore.css"] %}
+
+{% block extrabody %}
+
+<nav id="explore-links">
+  <p class="caption"> Explore </p>
+
+
+<ul>
+  <li><a href="https://dask.pydata.org"> Home </a></li>
+  <li>
+    <a href="">Algorithms</a>
```

Comment:
Placeholder, or do you think this is the correct name? I'm struggling to think of a better one, perhaps `Interfaces`?